internal/core/items/text.rs

```diff
@@ -465,6 +465,8 @@ impl Item for TextInput {
                 if let Some(keycode) = event.text.chars().next() {
                     if keycode == key_codes::Return && !self.read_only() && self.single_line() {
                         Self::FIELD_OFFSETS.accepted.apply_pin(self).call(&());
+                        WindowInner::from_pub(window_adapter.window())
+                            .request_close_virtual_keyboard();
```

Comment:
Hmm, is this needed?

If the accepted callback is handled, then it should result in a transfer of focus IMO, which would hide the virtual keyboard. If not, then the keyboard should remain visible, shouldn't it? (there's no way of bringing it back)

Reply:
I think you right, it should not be closed.